examples/simple/Error_Codes.csv

```diff
@@ -48,109 +48,108 @@ UB-CEER1,Trying to read through a null pointer.,"6.3.2.1:1, J.2:1 item 19",Undef
+CV-CEB11,Both bitwise inclusive OR operands must have integer type.,6.5.12:2,Constraint Violation
+CV-CEM3,Attempting to access member but it does not exist.,6.5.2.3:1,Constraint Violation
 CV-CER1,Unary '&' operator applied to non-lvalue.,6.5.3.2:1,Constraint Violation
 CV-CER2,Unary '&' operator applied to a value with register storage class.,6.5.3.2:1,Constraint Violation
 CV-CERL5,Directly comparing an integer type with a pointer type.,6.5.8:2,Constraint Violation
 CV-00035,Trying to write through a const lvalue.,6.5.16.1:1,Constraint Violation
 CV-CTM2,Variable-length array with file scope.,6.7.6.2:2,Constraint Violation
-CV-EFNC2,incompatible types in function call arguments.,6.5.16.1:1,Constraint Violation
-CV-EFNC3,incompatible pointer types in function call arguments.,6.5.16.1:1,Constraint Violation
-CV-TDG2,declaration for appears with multiple storage class specifiers.,6.7.1:2,Constraint Violation
-CV-TDG4,declaration appears with invalid storage class specifiers.,6.9:2,Constraint Violation
-CV-TDG5,function parameter appears with storage class other than register.,6.7.6.3:2,Constraint Violation
-CV-TDG8,declaration has initializer but incomplete type.,6.7.9:3,Constraint Violation
-CV-TDG10,conflicting types for identifier.,6.7:4,Constraint Violation
+CV-EFNC2,Incompatible types in function call arguments.,6.5.16.1:1,Constraint Violation
+CV-EFNC3,Incompatible pointer types in function call arguments.,6.5.16.1:1,Constraint Violation
+CV-TDG2,Declaration for appears with multiple storage class specifiers.,6.7.1:2,Constraint Violation
+CV-TDG4,Declaration appears with invalid storage class specifiers.,6.9:2,Constraint Violation
+CV-TDG5,Function parameter appears with storage class other than register.,6.7.6.3:2,Constraint Violation
+CV-TDG8,Declaration has initializer but incomplete type.,6.7.9:3,Constraint Violation
+CV-TDG10,Conflicting types for identifier.,6.7:4,Constraint Violation
 CV-TDG13,Redefinition of symbol:,6.9:3,Constraint Violation
 CV-TDL1,Extern declaration follows one with no linkage,6.7:3,Constraint Violation
 CV-TDL2,Extern declaration appears at block scope with an initializer.,6.7.9:5,Constraint Violation
 CV-TDL3,Extern declaration with incompatible type,6.7:4,Constraint Violation
 CV-TDL4,Redeclaration with no linkage,6.7:3,Constraint Violation
 CV-CDI4,Attempting to initialize nonexistent member.,6.7.9:2,Constraint Violation
-CV-TDG14,redefinition of symbol with external linkage,"6.9:5""",Constraint Violation
-CV-TDG15,"redefinition of struct, union, or enum",6.7.2.3:1,Constraint Violation
-CV-TDG18,declaration for appears with invalid storage class specifiers.,6.7.1:3,Constraint Violation
-CV-TDG16,incompatible redeclaration of symbol with internal or external linkage,6.7:4,Constraint Violation
+CV-TDG15,"Redefinition of struct, union, or enum",6.7.2.3:1,Constraint Violation
+CV-TDG18,Declaration for appears with invalid storage class specifiers.,6.7.1:3,Constraint Violation
```

Comment:
CV-TDG2 should occur when there is more than one storage class specifier in the identifier declaration, while TDG18 should occurs when the identifier declaration has an invalid storage class specifier. @dwightguth can you confirm if the intended usage in the semantics is such?